maven-plugins/stager-maven-plugin/src/test/java/io/helidon/build/maven/stager/StagingTaskTest.java

```diff
@@ -96,4 +87,66 @@ public String describe(Path dir, Map<String, String> variables) {
             return "test";
         }
     }
+
+    private static final class ContextTestImpl implements StagingContext {
+
+        @Override
+        public void unpack(Path archive, Path target, String excludes, String includes) {
+
+        }
+
+        @Override
+        public void archive(Path directory, Path target, String excludes, String includes) {
+
+        }
+
+        @Override
+        public Path resolve(String path) {
+            return null;
+        }
+
+        @Override
+        public Path resolve(ArtifactGAV gav) {
+            return null;
+        }
+
+        @Override
+        public Path createTempDirectory(String prefix) throws IOException {
+            return null;
+        }
+
+        @Override
+        public void logInfo(String msg, Object... args) {
```

Comment:
Let's update StagingContext and add default implementation for all the methods below (log*, submit, await termination)